openfisca_us/tests/policy/baseline/calcfunctions/gainstax.yaml

```diff
@@ -0,0 +1,261 @@
+- name: GainsTax unit test 0
```

Comment:
Is this generated from the script with true CPS tax units? If so could you add a comment indicating as such?

It may also be helpful to include the CPS record ID in the test name or a comment.